test/services/fetchers/binancePriceMultiFetcher.test.ts

```diff
@@ -15,12 +14,9 @@ describe.only('BinancePriceMultiFetcher', () => {
   let settings: Settings;
   let binancePriceMultiFetcher: BinancePriceMultiFetcher;
 
-  const params: InputParams[] = [
-    {id: 'BTC', currency: 'USDT'},
-    {id: 'ETH', currency: 'USDT'},
-  ];
+  const params: InputParams[] = [{symbol: 'BTCUSDT'}, {symbol: 'ETHUSDT'}];
```

Comment:
will `USDTBTC` work? if not we need to add option to reverse the price so we can have it both ways.

Reply:
Hello Dariousz, the price USDTBTC is not in available in the Binance API.

Would you like that we check in the case we don't find it to try to fetch `BTCUSDT` and then we invert it?

Reply:
yes, if not avilable, just revert ir